### PR TITLE
Updates IPFS from 0.8 to 0.9.

### PR DESCRIPTION
**Pull request by johnnymatthews**
Waiting for the Go IPFS 0.9 release to go live. When it does, we still need to run the API docs generation script against the new Go-IPFS version, and merge the new docs into this PR. But once that's done, we can merge this :)

ciskip

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/concepts/persistence.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://www.axel.org/blog/2019/07/23/qa-with-the-developers-of-axel-ipfs/
- This phrase is duplicated. You should probably leave only “Garbage Collection”.
  ```
  ...ou wish to keep on IPFS indefinitely. # Garbage Collection Garbage collectio...
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ... the overall lifecycle of data in IPFS, check out the course from IPFS Camp ...
                                             ^^^^^^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ... delete or garbage-collect your data on your own computer. Some available pi...
                                             ^^^^^^^^
  ```
- ‘Pinata’ is an imported foreign expression, which originally has a diacritic. Consider using “piñata”
  ```
  ... new window) Infura  (opens new window) Pinata  (opens new window) Temporal ...
                                             ^^^^^^
  ```
### `docs/concepts/privacy.md`:
- The following link returned HTTP status code 404 (Not Found):
  https://github.com/ipfs/go-ipfs/blob/v0.9.0/docs/experimental-features.md
- ‘by design’ might be wordy. Consider a shorter alternative.
  ```
  ...peer data storage and delivery, IPFS is by design a public network: Nodes pa...
                                             ^^^^^^^^^
  ```
- Consider adding a comma but only if you want to indicate a pause in casual speech.
  ```
  ...iding which CIDs is publicly available. So why doesn't the IPFS protocol its...
                                             ^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ...ried about the implications of this for your own personal use case, it's wor...
                                             ^^^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...ou've explicitly pinned or added to MFS in order to make them consistently a...
                                             ^^^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...ble over IPFS. # Using a public gateway Using a public IPFS gateway is one w...
                                             ^^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ...eady be an open-source codebase to help your own project achieve this. # Enc...
                                             ^^^^^^^^
  ```
- The word ‘caveat’ is a legal term. To make your text as clear as possible to all readers, do not use this foreign term unless it is used with its legal meaning. Possible alternatives are “caution” or “warning”.
  ```
  ... the ability to decrypt it. There's one caveat to keep in mind here: While t...
                                             ^^^^^^
  ```
- ‘Pinata’ is an imported foreign expression, which originally has a diacritic. Consider using “piñata”
  ```
  ...n. (For more details, this article from Pinata  (opens new window) may be he...
                                             ^^^^^^
  ```
### `docs/install/command-line.md`:
- Possibly missing comma found.
  ```
  ... Desktop can do, but at a more granular level since you can specify which co...
                                             ^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...n.  # System requirements IPFS requires 512MiB of memory and can run an IPFS...
                                             ^^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...ing. A base installation takes up about 12MB of disk space, and the default ...
                                             ^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ... default maximum disk storage is set to 10GB. # Official distributions The I...
                                             ^^^^
  ```
- Make sure that the singular noun after the number ‘9.0’ is correct. Did you mean “folders”?
  ```
  ...re handy.  Move into the go-ipfs_v0.9.0 folder and check that the ipfs.exe w...
                                             ^^^^^^
  ```
- The word ‘install’ is not a noun.
  ```
  ...ove into the go-ipfs folder and run the install script:  Check that IPFS ins...
                                             ^^^^^^^
  ```
- The word ‘install’ is not a noun.
  ```
  ...ove into the go-ipfs folder and run the install script:  Test that IPFS has ...
                                             ^^^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...ons and services on top of the network! Check out the Command-line- quicks s...
                                             ^^^^^^^^^
  ```
- After ‘the’, the verb ‘Initialize’ doesn’t fit. Is ‘Initialize’ spelled correctly? If ‘Initialize’ is the first word in a compound adjective, use a hyphen between the two words. Using the verb ‘Initialize’ as a noun may be non-standard.
  ```
  ...quicks start guide and jump straight to the Initialize the repository sectio...
                                             ^^^^^^^^^^^^^^
  ```
### `docs/install/recent-releases.md`:
- The following link returned HTTP status code 404 (Not Found):
  https://github.com/ipfs/go-ipfs/releases/tag/v0.9.0

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/concepts/persistence.md`:
- This phrase is duplicated. You should probably leave only “Garbage Collection”.
  ```
  ...ou wish to keep on IPFS indefinitely. # Garbage Collection Garbage collectio...
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ... the overall lifecycle of data in IPFS, check out the course from IPFS Camp ...
                                             ^^^^^^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ... delete or garbage-collect your data on your own computer. Some available pi...
                                             ^^^^^^^^
  ```
- ‘Pinata’ is an imported foreign expression, which originally has a diacritic. Consider using “piñata”
  ```
  ... new window) Infura  (opens new window) Pinata  (opens new window) Temporal ...
                                             ^^^^^^
  ```
### `docs/concepts/privacy.md`:
- The following link returned HTTP status code 404 (Not Found):
  https://github.com/ipfs/go-ipfs/blob/v0.9.0/docs/experimental-features.md
- ‘by design’ might be wordy. Consider a shorter alternative.
  ```
  ...peer data storage and delivery, IPFS is by design a public network: Nodes pa...
                                             ^^^^^^^^^
  ```
- Consider adding a comma but only if you want to indicate a pause in casual speech.
  ```
  ...iding which CIDs is publicly available. So why doesn't the IPFS protocol its...
                                             ^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ...ried about the implications of this for your own personal use case, it's wor...
                                             ^^^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...ou've explicitly pinned or added to MFS in order to make them consistently a...
                                             ^^^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...ble over IPFS. # Using a public gateway Using a public IPFS gateway is one w...
                                             ^^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ...eady be an open-source codebase to help your own project achieve this. # Enc...
                                             ^^^^^^^^
  ```
- The word ‘caveat’ is a legal term. To make your text as clear as possible to all readers, do not use this foreign term unless it is used with its legal meaning. Possible alternatives are “caution” or “warning”.
  ```
  ... the ability to decrypt it. There's one caveat to keep in mind here: While t...
                                             ^^^^^^
  ```
- ‘Pinata’ is an imported foreign expression, which originally has a diacritic. Consider using “piñata”
  ```
  ...n. (For more details, this article from Pinata  (opens new window) may be he...
                                             ^^^^^^
  ```
### `docs/install/command-line.md`:
- Possibly missing comma found.
  ```
  ... Desktop can do, but at a more granular level since you can specify which co...
                                             ^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...n.  # System requirements IPFS requires 512MiB of memory and can run an IPFS...
                                             ^^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...ing. A base installation takes up about 12MB of disk space, and the default ...
                                             ^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ... default maximum disk storage is set to 10GB. # Official distributions The I...
                                             ^^^^
  ```
- Make sure that the singular noun after the number ‘9.0’ is correct. Did you mean “folders”?
  ```
  ...re handy.  Move into the go-ipfs_v0.9.0 folder and check that the ipfs.exe w...
                                             ^^^^^^
  ```
- The word ‘install’ is not a noun.
  ```
  ...ove into the go-ipfs folder and run the install script:  Check that IPFS ins...
                                             ^^^^^^^
  ```
- The word ‘install’ is not a noun.
  ```
  ...ove into the go-ipfs folder and run the install script:  Test that IPFS has ...
                                             ^^^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...ons and services on top of the network! Check out the Command-line- quicks s...
                                             ^^^^^^^^^
  ```
- After ‘the’, the verb ‘Initialize’ doesn’t fit. Is ‘Initialize’ spelled correctly? If ‘Initialize’ is the first word in a compound adjective, use a hyphen between the two words. Using the verb ‘Initialize’ as a noun may be non-standard.
  ```
  ...quicks start guide and jump straight to the Initialize the repository sectio...
                                             ^^^^^^^^^^^^^^
  ```
### `docs/install/recent-releases.md`:
- The following link returned HTTP status code 404 (Not Found):
  https://github.com/ipfs/go-ipfs/releases/tag/v0.9.0

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/concepts/persistence.md`:
- This phrase is duplicated. You should probably leave only “Garbage Collection”.
  ```
  ...ou wish to keep on IPFS indefinitely. # Garbage Collection Garbage collectio...
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ... the overall lifecycle of data in IPFS, check out the course from IPFS Camp ...
                                             ^^^^^^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ... delete or garbage-collect your data on your own computer. Some available pi...
                                             ^^^^^^^^
  ```
- ‘Pinata’ is an imported foreign expression, which originally has a diacritic. Consider using “piñata”
  ```
  ... new window) Infura  (opens new window) Pinata  (opens new window) Temporal ...
                                             ^^^^^^
  ```
### `docs/concepts/privacy.md`:
- ‘by design’ might be wordy. Consider a shorter alternative.
  ```
  ...peer data storage and delivery, IPFS is by design a public network: Nodes pa...
                                             ^^^^^^^^^
  ```
- Consider adding a comma but only if you want to indicate a pause in casual speech.
  ```
  ...iding which CIDs is publicly available. So why doesn't the IPFS protocol its...
                                             ^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ...ried about the implications of this for your own personal use case, it's wor...
                                             ^^^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...ou've explicitly pinned or added to MFS in order to make them consistently a...
                                             ^^^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...ble over IPFS. # Using a public gateway Using a public IPFS gateway is one w...
                                             ^^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ...eady be an open-source codebase to help your own project achieve this. # Enc...
                                             ^^^^^^^^
  ```
- The word ‘caveat’ is a legal term. To make your text as clear as possible to all readers, do not use this foreign term unless it is used with its legal meaning. Possible alternatives are “caution” or “warning”.
  ```
  ... the ability to decrypt it. There's one caveat to keep in mind here: While t...
                                             ^^^^^^
  ```
- ‘Pinata’ is an imported foreign expression, which originally has a diacritic. Consider using “piñata”
  ```
  ...n. (For more details, this article from Pinata  (opens new window) may be he...
                                             ^^^^^^
  ```
### `docs/install/command-line.md`:
- Possibly missing comma found.
  ```
  ... Desktop can do, but at a more granular level since you can specify which co...
                                             ^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...n.  # System requirements IPFS requires 512MiB of memory and can run an IPFS...
                                             ^^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...ing. A base installation takes up about 12MB of disk space, and the default ...
                                             ^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ... default maximum disk storage is set to 10GB. # Official distributions The I...
                                             ^^^^
  ```
- Make sure that the singular noun after the number ‘9.0’ is correct. Did you mean “folders”?
  ```
  ...re handy.  Move into the go-ipfs_v0.9.0 folder and check that the ipfs.exe w...
                                             ^^^^^^
  ```
- The word ‘install’ is not a noun.
  ```
  ...ove into the go-ipfs folder and run the install script:  Check that IPFS ins...
                                             ^^^^^^^
  ```
- The word ‘install’ is not a noun.
  ```
  ...ove into the go-ipfs folder and run the install script:  Test that IPFS has ...
                                             ^^^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...ons and services on top of the network! Check out the Command-line- quicks s...
                                             ^^^^^^^^^
  ```
- After ‘the’, the verb ‘Initialize’ doesn’t fit. Is ‘Initialize’ spelled correctly? If ‘Initialize’ is the first word in a compound adjective, use a hyphen between the two words. Using the verb ‘Initialize’ as a noun may be non-standard.
  ```
  ...quicks start guide and jump straight to the Initialize the repository sectio...
                                             ^^^^^^^^^^^^^^
  ```
### `docs/install/recent-releases.md`:
- The following link returned HTTP status code 404 (Not Found):
  https://github.com/ipfs/go-ipfs/releases/tag/v0.9.0

File: docs/install/command-line.md [hunk 1]
```diff
@@ -69,21 +69,21 @@ The IPFS team manages the [dist.ipfs.io website](https://dist.ipfs.io/) to help
...
    ```
 
 ### macOS
 
 1. Download the macOS binary from [`dist.ipfs.io`](https://dist.ipfs.io/#go-ipfs).
 
    ```bash
-   wget https://dist.ipfs.io/go-ipfs/v0.8.0/go-ipfs_v0.8.0_darwin-amd64.tar.gz
+   wget https://dist.ipfs.io/go-ipfs/v0.9.0/go-ipfs_v0.9.0_darwin-amd64.tar.gz
    ```
```
Comment (aschmahmann) [inline docs/install/command-line.md:81]: We may want to add a comment here about how you can grab the ones built for Darwin ARM64 (i.e. the M1 hardware) at https://dist.ipfs.io/go-ipfs/v0.9.0/go-ipfs_v0.9.0_darwin-arm64.tar.gz

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/concepts/persistence.md`:
- This phrase is duplicated. You should probably leave only “Garbage Collection”.
  ```
  ...ou wish to keep on IPFS indefinitely. # Garbage Collection Garbage collectio...
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ... the overall lifecycle of data in IPFS, check out the course from IPFS Camp ...
                                             ^^^^^^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ... delete or garbage-collect your data on your own computer. Some available pi...
                                             ^^^^^^^^
  ```
- ‘Pinata’ is an imported foreign expression, which originally has a diacritic. Consider using “piñata”
  ```
  ... new window) Infura  (opens new window) Pinata  (opens new window) Temporal ...
                                             ^^^^^^
  ```
### `docs/concepts/privacy.md`:
- ‘by design’ might be wordy. Consider a shorter alternative.
  ```
  ...peer data storage and delivery, IPFS is by design a public network: Nodes pa...
                                             ^^^^^^^^^
  ```
- Consider adding a comma but only if you want to indicate a pause in casual speech.
  ```
  ...iding which CIDs is publicly available. So why doesn't the IPFS protocol its...
                                             ^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ...ried about the implications of this for your own personal use case, it's wor...
                                             ^^^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...ou've explicitly pinned or added to MFS in order to make them consistently a...
                                             ^^^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...ble over IPFS. # Using a public gateway Using a public IPFS gateway is one w...
                                             ^^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ...eady be an open-source codebase to help your own project achieve this. # Enc...
                                             ^^^^^^^^
  ```
- The word ‘caveat’ is a legal term. To make your text as clear as possible to all readers, do not use this foreign term unless it is used with its legal meaning. Possible alternatives are “caution” or “warning”.
  ```
  ... the ability to decrypt it. There's one caveat to keep in mind here: While t...
                                             ^^^^^^
  ```
- ‘Pinata’ is an imported foreign expression, which originally has a diacritic. Consider using “piñata”
  ```
  ...n. (For more details, this article from Pinata  (opens new window) may be he...
                                             ^^^^^^
  ```
### `docs/install/command-line.md`:
- Possibly missing comma found.
  ```
  ... Desktop can do, but at a more granular level since you can specify which co...
                                             ^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...n.  # System requirements IPFS requires 512MiB of memory and can run an IPFS...
                                             ^^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...ing. A base installation takes up about 12MB of disk space, and the default ...
                                             ^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ... default maximum disk storage is set to 10GB. # Official distributions The I...
                                             ^^^^
  ```
- Make sure that the singular noun after the number ‘9.0’ is correct. Did you mean “folders”?
  ```
  ...re handy.  Move into the go-ipfs_v0.9.0 folder and check that the ipfs.exe w...
                                             ^^^^^^
  ```
- The word ‘install’ is not a noun.
  ```
  ...ove into the go-ipfs folder and run the install script:  Check that IPFS ins...
                                             ^^^^^^^
  ```
- The word ‘install’ is not a noun.
  ```
  ...ove into the go-ipfs folder and run the install script:  Test that IPFS has ...
                                             ^^^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...ons and services on top of the network! Check out the Command-line- quicks s...
                                             ^^^^^^^^^
  ```
- After ‘the’, the verb ‘Initialize’ doesn’t fit. Is ‘Initialize’ spelled correctly? If ‘Initialize’ is the first word in a compound adjective, use a hyphen between the two words. Using the verb ‘Initialize’ as a noun may be non-standard.
  ```
  ...quicks start guide and jump straight to the Initialize the repository sectio...
                                             ^^^^^^^^^^^^^^
  ```
- Check of `docs/install/recent-releases.md` came back clean!

File: docs/install/command-line.md (hunk 1 above)
Comment (johnnymatthews) [reply docs/install/command-line.md:85]: Good call. I've added a call out to tell users how to install Go-IPFS on M1-based Macs.

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/concepts/persistence.md`:
- This phrase is duplicated. You should probably leave only “Garbage Collection”.
  ```
  ...ou wish to keep on IPFS indefinitely. # Garbage Collection Garbage collectio...
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ... the overall lifecycle of data in IPFS, check out the course from IPFS Camp ...
                                             ^^^^^^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ... delete or garbage-collect your data on your own computer. Some available pi...
                                             ^^^^^^^^
  ```
- ‘Pinata’ is an imported foreign expression, which originally has a diacritic. Consider using “piñata”
  ```
  ... new window) Infura  (opens new window) Pinata  (opens new window) Temporal ...
                                             ^^^^^^
  ```
### `docs/concepts/privacy.md`:
- ‘by design’ might be wordy. Consider a shorter alternative.
  ```
  ...peer data storage and delivery, IPFS is by design a public network: Nodes pa...
                                             ^^^^^^^^^
  ```
- Consider adding a comma but only if you want to indicate a pause in casual speech.
  ```
  ...iding which CIDs is publicly available. So why doesn't the IPFS protocol its...
                                             ^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ...ried about the implications of this for your own personal use case, it's wor...
                                             ^^^^^^^^
  ```
- Consider a shorter alternative to avoid wordiness.
  ```
  ...ou've explicitly pinned or added to MFS in order to make them consistently a...
                                             ^^^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...ble over IPFS. # Using a public gateway Using a public IPFS gateway is one w...
                                             ^^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ...eady be an open-source codebase to help your own project achieve this. # Enc...
                                             ^^^^^^^^
  ```
- The word ‘caveat’ is a legal term. To make your text as clear as possible to all readers, do not use this foreign term unless it is used with its legal meaning. Possible alternatives are “caution” or “warning”.
  ```
  ... the ability to decrypt it. There's one caveat to keep in mind here: While t...
                                             ^^^^^^
  ```
- ‘Pinata’ is an imported foreign expression, which originally has a diacritic. Consider using “piñata”
  ```
  ...n. (For more details, this article from Pinata  (opens new window) may be he...
                                             ^^^^^^
  ```
### `docs/install/command-line.md`:
- Possibly missing comma found.
  ```
  ... Desktop can do, but at a more granular level since you can specify which co...
                                             ^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...n.  # System requirements IPFS requires 512MiB of memory and can run an IPFS...
                                             ^^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...ing. A base installation takes up about 12MB of disk space, and the default ...
                                             ^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ... default maximum disk storage is set to 10GB. # Official distributions The I...
                                             ^^^^
  ```
- Make sure that the singular noun after the number ‘9.0’ is correct. Did you mean “folders”?
  ```
  ...re handy.  Move into the go-ipfs_v0.9.0 folder and check that the ipfs.exe w...
                                             ^^^^^^
  ```
- The word ‘install’ is not a noun.
  ```
  ...ove into the go-ipfs folder and run the install script:  Check that IPFS ins...
                                             ^^^^^^^
  ```
- The word ‘install’ is not a noun.
  ```
  ...ove into the go-ipfs folder and run the install script:  Test that IPFS has ...
                                             ^^^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...ons and services on top of the network! Check out the Command-line- quicks s...
                                             ^^^^^^^^^
  ```
- After ‘the’, the verb ‘Initialize’ doesn’t fit. Is ‘Initialize’ spelled correctly? If ‘Initialize’ is the first word in a compound adjective, use a hyphen between the two words. Using the verb ‘Initialize’ as a noun may be non-standard.
  ```
  ...quicks start guide and jump straight to the Initialize the repository sectio...
                                             ^^^^^^^^^^^^^^
  ```
- Check of `docs/install/recent-releases.md` came back clean!

**Pull request merged**